backend/routes/complaints.routes.ts

```diff
@@ -0,0 +1,35 @@
+import express from "express";
+const router = express.Router();
+const Complaint = require("../models/Complaint");
+
+router.get("/complaints", async (req, res) => {
+	try {
+		const result = await Complaint.find({});
+		res.json(result);
```

Comment:
this should have an `await`

Reply:
You're not sending back a request or anything. You have to respond back to the application

Reply:
I dont get this one, there is an await on line 7 and i do get the complaints on the frontend. can you clarify?

Reply:
Whoops! You're right. You can disregard this comment 😊